internal/output/output.go

```diff
@@ -33,6 +33,12 @@ func BuildFor(o *taskfile.Output) (Output, error) {
 			Begin: o.Group.Begin,
 			End:   o.Group.End,
 		}, nil
+	case "group_error_only":
```

Comment:
After some reflection, I think that we should probably follow the existing pattern for `begin/end` and allow an additional option on `group`, instead of having a separate option. Would be like this:

```yaml
output:
  group:
    begin: '::group::{{.TASK}}'
    end: '::endgroup::'
    error_only: true
```

`begin` and `end` would be optional, of course, if the users wants only `error_only`.

What do you think?

Reply:
done